src/avatar/__tests__/__snapshots__/Avatar.js.snap

```diff
@@ -61,7 +61,6 @@ exports[`Avatar Component should render avatar without height 1`] = `
 
 exports[`Avatar Component should render avatar without width 1`] = `
 <View
-  activeOpacity={0.7}
```

Comment:
the activeOpacity prop is only valid when the component prop is a TouchableOpacity.  In this test case, it's the default View, so the test never was testing a TouchableOpacity case.

I updated the Avatar component and this snapshot because web shows warnings when unknown props are passed through to the dom.

Reply:
the activeOpacity prop is only valid when the component prop is a TouchableOpacity. In this test case, it's the default View, so the test never was testing a TouchableOpacity case.

I updated the Avatar component to strip this prop when appropriate because web shows warnings when unknown props are passed through to the dom.

Reply:
Oic, sorry.  It's scope creep for this PR obviously, but the snapshots for the different cases probably need to be added.